Improve DBus man page

exquo opened the pull request:
- Expand synopsis
- Add examples
- Fix typos

exquo commented:
Also, I'm not sure what `setContacts` refers to here:
https://github.com/AsamK/signal-cli/commit/caf16231aaf7550cedf74c66f5ddfa9a82b7bb88#diff-9ccadddc1e2765302bd0becc272f14ffe202fa4a08e64bf7ea3c28aabe9b3e66R129

AsamK commented on man/signal-cli-dbus.5.adoc:
```diff
@@ -11,10 +11,11 @@ DBus API for signal-cli - A commandline and dbus interface for the Signal messen
 == Synopsis
 
-*signal-cli* [--config CONFIG] daemon --dbus-system
+*signal-cli* [--verbose] [--config CONFIG] [-u USERNAME] [-o {plain-text,json}] [--dbus-system] daemon
```
for system dbus it's actually `--system` after `daemon` :sweat_smile: 
`signal-cli daemon [--system]`

exquo replied:
I see, thanks! Should have checked the 'main' man page :). (I've never used the system bus myself)

AsamK commented on man/signal-cli-dbus.5.adoc:
```diff
@@ -198,6 +199,19 @@ MessageReceived(timestamp<x>, sender<s>, groupId<ay>, message<s>, attachments<as
+== Examples
+
+Send a text message (without attachment) to a contact::
+dbus-send --print-reply --type=method_call --dest="org.asamk.Signal"
```
I think the command should be in a single line

exquo replied:
Battling vim's auto-formatting :)

AsamK reviewed (commented):
Thanks a lot!
I think the `setContacts` should be `setContactName`

exquo commented:
> I think the setContacts should be setContactName

Makes sense. I've also added that it might be copied over from phone's contacts for a linked account.

exquo commented:
I've pushed another commit addressing your comments - feel free to squash with the original. 

BTW, do you have a 'recommended' way of updating PRs? Like for the PR author to do a force-push into the original branch? Or do you prefer to get separate commits (with an option to squash them later)?

AsamK commented:
I normally use the squash button to merge PRs, so it doesn't really matter. I don't mind force pushes to the branch.
If the PR is split into logical commits, that make the changes easier to understand I'd keep the commits, though it might be better to open separate PRs in that case.

exquo commented:
I see, thank you! Might be useful to know in the future.
